cypress/integration/basic/group_rules.spec.js

```diff
@@ -0,0 +1,86 @@
+describe('Test that group rules can be created', () => {
```

Comment:
Update describe() and it() language as explained here https://github.com/dssit/roles-management/pull/349